### ISSUE TITLE
Pokerstars SnGs stats

### ISSUE THREAD

**Issue by sxe** [philroberts/FPDB-for-OSX#46]
Hi, 
i imported a couple of sngs and some random tournaments. That worked without problems as far as i can tell but the "Tourney Stas" window does not seem to show correct data.

The # column for example shows 67 what might be correct but so does the Rank? column. What does that mean? 
The Spent column seems to be correct as well but ITM%, the rank colums (1st,2sr ..), Won, ROI%, $/Tour are all empty or zero. 
Any chance to fix that? Maybe i forgot something?

I am using teh latest fpdb version and pokerstars.

Thx

**Comment by philroberts** [comment]
Yeah unfortunately tourney support is not very good.  I don't play them myself so it's not really on my radar.

**Comment by sxe** [comment]
Wow that was fast. Any chance you could point me to the correct files to alter. I am a software developer myself, maybe i can have a look myself in the future if it's not too much work.

**Comment by philroberts** [comment]
If you start at GuiTourneyPlayerStats.py you can probably follow the spaghetti where you need to go.  :)

Also I have a vague recollection that you need some kind of tournament summary file for the ITM/ROI things, not just the hand history.  But I could be mistaken on that.

**Comment by sxe** [comment]
ok thx. 

I got some better results now for what ever reason. 
I copied all hh files to an other folder and pointed the "bulk import" location to that folder (fresh database file). Now the Tourney stats look kinda ok. All the columns are filled (besids rank?) and at least make sense without me having validated them in detail. I wounder what made the difference. The only difference is the different file location as far as i can tell. 

*edit: Ah, the difference is that i only imported the tournament summary files now. Which does what i wanted in the first place. Pokerstars saves a detailed tournament file in the normal hh file location which seems to confuse fpdb. So just importing the summary files seems to work.

*edit2: But this also means no hud of course, cause you have no detailed player stas in the database.

**Comment by philroberts** [comment]
Hmm that's interesting.  Maybe the order of import matters?  I feel like this is something that could be fixed without too much work.  As a workaround you could have one database for the summary stats and another one for the hands/HUD.

**Comment by sxe** [comment]
Yeah having two databases would work i guess, but right now i am not that mich interested in hud stats. A way to get a summary of my results is all i need. Sadly knockout tourneys will not index correctly, cause there is no reference to the bounties in the tourney summary hhs.

**Comment by ChazDazzle** [comment]
Bounties will get picked up in the hand histories themselves. I'm not sure i understand why importing both the hands and the tournaments summaries didn't work. Could you explain that a bit more?

**Comment by sxe** [comment]
@ChazDazzle sure.

I made some screenshots to make it more easy to follow.

1. This is how my tourney stats look like when i just import the tourney summaries: http://i.imgur.com/6Ofqovp.png The stats here look ok but the bounties are missing.

2. This is how it looks like after i selected the normal hand history folder in the settings and clicked "bulk import" again:  http://i.imgur.com/9dQS7f5.png As you can see. The stats don't look correct anymore. The tourneys are split up differently and the bounties are still missing i guess.

3. Interestingly i get yet an other result when i delete the database and hit "bulk import" again, so all hhs (normal and summary) are imported at the same time. http://i.imgur.com/g0mDJkH.png

 Hope that helps. Let me know if you need the hhs or the logs.

Greetings

**Comment by ChazDazzle** [comment]
Could you expand out the first column in the last image?

It looks like the hand histories are adding information to the database to
split off 5 of the tournaments into a separate category. Otherwise
everything is the same just out of order.

I'll have to take a look at the kos to see where they should be pulled in.

**Comment by sxe** [comment]
What do you mean by "expand out"? If you mean, making the column wider, there is nothing there. The value is just "PokerStars" in all rows. Also, Won, ROI% and $/Tour is missing compared to the first image.

**Comment by ChazDazzle** [comment]
Ok, thanks. Can you try importing just the tournament summaries?

**Comment by sxe** [comment]
That's what i did in my first screenshot.

**Comment by ChazDazzle** [comment]
The first screenshot is tourney summaries only, correct? The second
screenshot is hands + tourney summaries. I think the third screenshot is
probably hand histories only. Since there's no results my guess is the
tournament summaries only get imported correctly if you do them on their
own.

**Comment by sxe** [comment]
1. Summaries only, yes.
2. First Summaries, then All
3. Yeah you are right. I imported the summaries again and now the last screen looks like the second one. :) I was wrongfully under the impression that the "bulk import" imports both the normal and the summary hhs at the same time. 

So we have two problems. The split of the turbos and the missing bounties. 

Thx for your time looking at this mate.

**Comment by ChazDazzle** [comment]
How many bounty tournaments would you say you played?

**Comment by ChazDazzle** [comment]
Also, i'll make sure to update this view to include rebuys, addons, and
bounties for the next version.

**Comment by sxe** [comment]
Sounds nice.

I played 4 bounty tourney. 

No Limit Hold'em $1.01 + $0.09 Zoom
No Limit Hold'em $2.01 + $0.19
No Limit Hold'em $0.50 + $0.05
No Limit Hold'em $1.01 + $0.09 Zoom

Let me know if you need some testing.
